test/stardog__lindas__test.py

```diff
@@ -878,7 +909,7 @@ def testConstructByGETinCSV_Unexpected(self):
         ct = result.info()["content-type"]
         assert True in [one in ct for one in _SPARQL_DESCRIBE_CONSTRUCT_POSSIBLE]
         results = result.convert()
-        self.assertEqual(type(results), bytes) # text/turtle
+        self.assertEqual(type(results), bytes)  # text/turtle
```

Comment:
Why do you add an extra space here?

Reply:
It's one of the PEP conventions, so automatically added by Black